test/sensor_simulator/gps.h

```diff
@@ -51,27 +51,26 @@ class Gps: public Sensor
 	~Gps();
 
 	void setData(const gps_message& gps);
-	void stepHeightByMeters(float hgt_change);
-	void stepHorizontalPositionByMeters(Vector2f hpos_change);
-	void setPositionRateNED(const Vector3f& rate) { _gps_pos_rate = rate; }
-	void setAltitude(int32_t alt);
-	void setLatitude(int32_t lat);
-	void setLongitude(int32_t lon);
+	void stepHeightByMeters(const float hgt_change);
```

Comment:
I don't think using the `const` keyword in front of a native type in the function declaration is a good thing. It's anyway passing a copy, so the user doesn't care. You can however leave it in the definition to make sure you're not changing it in the function.